apps/src/applab/applab.js

```diff
@@ -73,9 +73,11 @@ var outputApplabConsole = errorHandler.outputApplabConsole;
 var outputError = errorHandler.outputError;
 var ErrorLevel = errorHandler.ErrorLevel;
 
+// TODO : should we just be exposing config?
```

Comment:
Didn't mean to leave this TODO here, but we *could* expose config, and then stop exposing level/skin/dropletConfigBlocks (and maybe copyrightRights) because they are all members of config. Cleaner in that there are fewer objects exposed.